doc/admin-guide/files/ssl_multicert.config.en.rst

```diff
@@ -93,6 +93,13 @@ ssl_ca_name=FILENAME (optional)
   the certificate chain. *FILENAME* is resolved relative to the
   :ts:cv:`proxy.config.ssl.CA.cert.path` configuration variable.
 
+ssl_ocsp_name=FILENAME (optional)
+  The name of the file containing the prefetched OCSP stapling response
+  for this certificate. This field can be omitted to let trafficserver
+  fetch OCSP responses dinamically, otherwise *FILENAME* is resolved
+  relative to the :ts:cv:`proxy.config.ssl.ocsp.response.path`
+  configuration variable.
```

Comment:
Perhaps, "Otherwise, when included, the administrator is responsible for updating the file's content."